app/assets/stylesheets/common/components/d-multi-select.scss

```diff
@@ -7,7 +7,7 @@
   gap: 0.25em;
   line-height: normal;
   box-sizing: border-box;
-  width: 200px;
+  width: 300px;
```

Comment:
same not directly related but using this component is showing that 300px would be better.